cclient/BUILD

```diff
@@ -1,28 +1,28 @@
 cc_library(
     name = "api",
-    srcs = glob([
-        "*.c",
-        "**/*.c",
-        "**/*.h",
-    ]),
+    srcs = [
+        "iota_api.c",
+        "iota_api.h",
+    ],
     hdrs = ["iota_api.h"],
     visibility = ["//visibility:public"],
     deps = [
-        ":service",
+        "//cclient:service",
         "//cclient/http:beast",
     ],
 )
 
 cc_library(
     name = "service",
     srcs = [
+        "service.c",
         "service.h",
```

Comment:
Header in the sources ?

Reply:
https://docs.bazel.build/versions/master/be/c-cpp.html#cc_binary.srcs